src/components/reusable/codeView/blockCodeView/blockCodeView.scss

```diff
@@ -196,7 +196,7 @@ $border-radius: 4px;
                 line-height: 1.6;
                 max-width: 75%;
                 box-sizing: border-box;
-                white-space: pre;
+                white-space: break-spaces;
```

Comment:
Do we actually want code to wrap lines? I was thinking more along the lines of `overflow-x: auto` for code blocks.

Reply:
we can do either. i have it set to use `overflow-x: auto` on the highlight.js implementation PR. 

but, yes, i feel like we should leave most of the actual formatting to the user, so overflow-x might be better

Reply:
✅